dashboard/apps/prototype/management/commands/cache.py

```diff
@@ -25,6 +26,7 @@ def generate(self):
         """
         generate cache
         """
+        cache_tools.METHOD_CACHE_WRITE_ONLY = True
```

Comment:
command line is run by celery, which is a separate environment and process from web app from the application, so it won't affect the web app. 

however, the indication is that in the celery process, this flag will be always `True` until restarted. do you think this may cause any problem?